src/renderers/webgl/WebGLPrograms.js

```diff
@@ -38,6 +38,8 @@ function WebGLPrograms( renderer, cubemaps, cubeuvmaps, extensions, capabilities
 	function getChannel( value ) {
 
 		if ( value === 1 ) return 'uv2';
```

Comment:
Maybe just `if ( value > 0 ) return 'uv' + ( value + 1 );`?

Reply:
The current code is more readable.